samples/snippets/cloud-client/src/main/java/privateca/MonitorCertificateAuthority.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package privateca;
+
+// [START privateca_monitor_ca_expiry]
+
+import com.google.cloud.monitoring.v3.AlertPolicyServiceClient;
+import com.google.cloud.monitoring.v3.NotificationChannelServiceClient;
+import com.google.monitoring.v3.AlertPolicy;
+import com.google.monitoring.v3.AlertPolicy.Condition;
+import com.google.monitoring.v3.AlertPolicy.Condition.MonitoringQueryLanguageCondition;
+import com.google.monitoring.v3.AlertPolicy.ConditionCombinerType;
+import com.google.monitoring.v3.NotificationChannel;
+import com.google.monitoring.v3.ProjectName;
+import java.io.IOException;
+
+public class MonitorCertificateAuthority {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "your-project-id";
+    createCaMonitoringPolicy(project);
+  }
+
+  // Creates a monitoring policy to notify 30 days before a managed CA expires.
+  public static void createCaMonitoringPolicy(String project) throws IOException {
+    /* Initialize client that will be used to send requests. This client only needs to be created
```

Comment:
Replace with "is". These are recommendations from the style guide: https://developers.google.com/style/tense. Let's make sure to follow them in the code comments as well.

Reply:
Don't have a strong opinion on this. We can choose to leave it as-is.